ta/crypt/include/user_ta_header_defines.h

```diff
@@ -36,9 +36,7 @@
 	and TA_FLAG_MULTI_SESSION */
 /* It is not possible to open it twice in a raw */
 #define TA_FLAGS		(TA_FLAG_USER_MODE | TA_FLAG_EXEC_DDR | \
-				TA_FLAG_SINGLE_INSTANCE | \
-				TA_FLAG_MULTI_SESSION \
-				)
+				TA_FLAG_MULTI_SESSION )
```

Comment:
I believe `TA_FLAG_MULTI_SESSION` has no effect when `TA_FLAG_SINGLE_INSTANCE` is not set. If so, you should remove it too.